src/Avalonia.Controls/TextBox.cs

```diff
@@ -208,6 +232,12 @@ public string Text
             }
         }
 
+        public string DisplayText
```

Comment:
I think there's probably an easier way to do this:

- Pass the text is passed to the `TextPresenter` like normal
- Add a `PasswordChar` property to `TextPresenter` and bind it
- In `TextPresenter.CreateFormattedText`, if `PasswordChar` is set, simply replace the text for display there

Would that work?

Reply:
Sure, will get this done

Reply:
Implemented as requested.